app/services/licences/fetch-licence.service.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+/**
+ * Fetches data needed for the bill run page which includes a summary for each bill linked to the bill run
+ * @module FetchLicenceService
+ */
+
+const LicenceModel = require('../../models/licence.model.js')
+
+/**
+ * Fetch the licence data from the water schema
+ *
+ * Was built to provide the data needed for the '/licences/{id}' page
+ *
+ * @param {string} id The UUID for the licence to fetch
+ *
+ * @returns {Promise<Object>} the data needed to populate the summary page
+ */
+async function go (id) {
+  const licence = await _fetchLicence(id)
+
+  return {
+    licence
+  }
```

Comment:
It might be not seeing the benefit (so feel free to push back!) But normally we just return the instance directly rather than wrapping it in a new object. Is there any reason not to do this?

```suggestion
  return licence
```